### DISCUSSION TITLE
Export collection to PDF too

### DISCUSSION THREAD

**Discussion by hermesalvesbr** [directus/directus#5642] [Feature Requests]
Directus allows me to export to json, csv and xml. To print, the option to generate the PDF is important. I propose to add the option to export as PDF as well.

![Captura de Tela 2021-05-12 às 17 24 31](https://user-images.githubusercontent.com/379850/118039641-ee2de580-b346-11eb-982a-516d6ee37e0f.png)

**Comment by rijkvanzanten** [comment]
What would an export to PDF look like? The problem is that a PDF is a visual representation of the data, which is very opinionated 🤔 Do we render just a table? Is it monospace? Does it have the project name and logo? etc etc

**Reply by hermesalvesbr** [reply]
Standard header, get logo and project name. Show the default table, it's enough.

**Reply by shadowcpy** [reply]
This should not be hardcoded, instead we could add the option for collection pdf presets that the user "designs" with some kind of template language, twig, or something

**Comment by hugopvl** [comment]
hello guys

is there any solution to export to pdf, or something that is possible to explore, including the images.

or already someone managed to export to excel, with images?

I need to be able to do this.

**Reply by shadowcpy** [reply]
You can press Ctrl + P on a page if you close the sidebar and then print to PDF, but this is very buggy and should probably be improved.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
